packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -19,6 +19,8 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/Cha
 +  catalog.cattle.io/experimental: true
 +  catalog.cattle.io/namespace: cattle-gatekeeper-system
 +  catalog.cattle.io/release-name: rancher-gatekeeper
++  catalog.cattle.io/auto-install: rancher-gatekeeper-crd=match
++  catalog.cattle.io/provides-gvr: configs.config.gatekeeper.sh/v1alpha1
```

Comment:
Lots of incorrect gvrs in here but I will follow up to that later, its unrelated.